src/TestHarness/Harness.cs

```diff
@@ -202,7 +202,7 @@ public Harness()
             };
 
```

Comment:
We should add some unit tests that deal with the track data, so that we exercise that piece of code.